src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java

```diff
@@ -161,6 +110,34 @@ public FileResource(FileService fileService, ResourceLoader resourceLoader, Lect
         return responseEntityForFilePath(FilePathService.getTempFilepath() + filename);
     }
 
+    /**
+     * POST /markdownFileUpload : Upload a new file for markdown.
+     *
+     * @param file The file to save
+     * @param keepFileName specifies if original file name should be kept
+     * @return The path of the file
+     * @throws URISyntaxException if response path can't be converted into URI
+     */
+    @PostMapping("/markdownFileUpload")
```

Comment:
just a small thing but maybe something like `markdown-file-upload`  instead of camelCase

Reply:
changed that now 👍